### PR TITLE
Launcher arguments

### PR DESCRIPTION
**Pull request by vorburger**

### CODE REVIEW

File: jetcd-core/pom.xml [hunk 1]
```diff
@@ -130,6 +130,11 @@
             <artifactId>commons-io</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.truth</groupId>
+            <artifactId>truth</artifactId>
```
[the remark this answers is not in the document]
Comment (fanminshi) [reply jetcd-core/pom.xml]: agreed. I think we should stick with assertJ.

[the remark this answers is not in the document]
Comment (vorburger) [reply jetcd-core/pom.xml]: @lburgazzoli @fanminshi OK, sure; I've just removed the Google Truth test lib. OK to merge?

File: jetcd-core/src/test/java/io/etcd/jetcd/launcher/test/EtcdClusterUsingTest.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+public class EtcdClusterUsingTest {
+
+  @Test
+  public void testUseEtcd() throws Exception {
+    try (EtcdCluster etcd = EtcdClusterFactory.buildCluster(getClass().getSimpleName(), 3, false, false)) {
+      etcd.start();
+      try (Client client = Client.builder().endpoints(etcd.getClientEndpoints()).build()) {
+        try (KV kvClient = client.getKVClient()) {
+
+          ByteSequence key = ByteSequence.from("test_key", UTF_8);
```
Comment (fanminshi) [inline jetcd-core/src/test/java/io/etcd/jetcd/launcher/test/EtcdClusterUsingTest.java]: is it possible to import `import static io.etcd.jetcd.TestUtil.bytesOf`?
and then do `ByteSequence KEY = bytesOf("test_key");`?

File: jetcd-core/src/test/java/io/etcd/jetcd/launcher/test/EtcdClusterUsingTest.java [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+          ByteSequence value = ByteSequence.from("test_value", UTF_8);
+          kvClient.put(key, value).get();
+
+          CompletableFuture<GetResponse> getFuture = kvClient.get(key);
+          GetResponse response = getFuture.get();
+          List<KeyValue> values = response.getKvs();
+          assertEquals(1, values.size());
+          KeyValue value1 = values.get(0);
+          assertEquals(value, value1.getValue());
```
Comment (fanminshi) [inline jetcd-core/src/test/java/io/etcd/jetcd/launcher/test/EtcdClusterUsingTest.java]: do you think that you can `import static org.assertj.core.api.Assertions.assertThat;` and use`assertThat` for assertion?

**Comment by fanminshi** [comment]
cc/ @yb172  could you take a look?

**Comment by yb172** [comment]
With assertions replaced to assertJ: looks good to me.

File: jetcd-core/src/test/java/io/etcd/jetcd/launcher/test/EtcdClusterUsingTest.java (hunk 2 above)
Comment (vorburger) [reply jetcd-core/src/test/java/io/etcd/jetcd/launcher/test/EtcdClusterUsingTest.java]: sure, done.

File: jetcd-core/src/test/java/io/etcd/jetcd/launcher/test/EtcdClusterUsingTest.java (hunk 3 above)
Comment (vorburger) [reply jetcd-core/src/test/java/io/etcd/jetcd/launcher/test/EtcdClusterUsingTest.java]: certainly - done.

**Comment by fanminshi** [comment]
lgtm

**Pull request merged**